src/shared/components/query/filteredSearch/field/dataTypeFilterField.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+import { FunctionComponent } from 'react';
+import { SearchClause } from 'shared/components/query/filteredSearch/SearchClause';
+
+import { FieldProps } from 'shared/components/query/filteredSearch/field/FilterFormField';
+import { ListPhrase } from 'shared/components/query/filteredSearch/Phrase';
+import { TableHeaderCellFilterIcon } from 'pages/studyView/table/TableHeaderCellFilterIcon';
+import { getDataTypeStudyOverlay } from 'pages/studyView/TableUtils';
+import { createQueryUpdate } from 'shared/components/query/filteredSearch/field/CheckboxFilterField';
+import styles from 'pages/studyView/table/tables.module.scss';
+
+export type dataTypeFilterField = {
```

Comment:
Let's define the type this way.

```suggestion
export type DataTypeFilterField = {
```

We should also name the file `DataTypeFilterField.tsx` for consistency